src/css/embed.scss

```diff
@@ -72,3 +72,17 @@ ul {
 .u-hidden {
     display: none;
 }
+
+.u-underline {
```

Comment:
This is very heavily based on the frontend class of the same name, but with a different default border due to the blue background. Not sure whether using the same name is sensible or not.